src/pageContent/PageContentLayout.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+import styles from "./pageContentLayout.module.scss";
+
+type Props = {
+  breadcrumbs?: React.ReactNode;
+  content: React.ReactNode;
+  sidebarContent: React.ReactNode;
+};
+
+export default function PageContentLayout({
```

Comment:
Should the `PageContentLayout` be renamed to something that describes that it has 70%-30% columns? E.g. `PageContentRightPanelLayout`. In future, we might have different layout styles. Sure this is something that could be taken care when we have such things, but it would be easier to update the client if the layout component name would not change during the update process.

Reply:
This component is an implementation detail. Consumers of the package can't import it.

These are the modules the package exports:

```tsx
// eslint-disable-next-line import/prefer-default-export
export { default as Page } from "./page/Page";

// eslint-disable-next-line import/prefer-default-export
export { default as PageContent } from "./pageContent/PageContent";
export type { Breadcrumb } from "./pageContent/types";

// eslint-disable-next-line import/prefer-default-export
export { default as ConfigProvider } from "./configProvider/ConfigProvider";
export { defaultConfig } from "./configProvider/ConfigProvider";
```

Importing other modules is not officially supported.

Reply:
I guess that the question could be repeated for `PageContent` though.

In its case, we could allow the layout to be changed with a prop that allows for the reimplementation of the alyout component, or allows the user to choose from a predefined list of layout components.